src/contextAndReducers/DataProvider.jsx

```diff
@@ -30,7 +30,6 @@ const DataProvider = ({ children }) => {
           const modifiedData = state.backUpData.filter(
             (item) => item.categoryName === action.payload
```

Comment:
You can directly destructure category name here. For e.g ```({categoryName})=>categoryName===action.payload```

Reply:
Yeah I know that purposely I didn't do it. Thanks for reviewing though